puppet.do.jenkins.io.tf

```diff
@@ -5,7 +5,7 @@ resource "digitalocean_ssh_key" "puppet_do_jenkins_io" {
 
 resource "digitalocean_droplet" "puppet_do_jenkins_io" {
   image       = "ubuntu-20-04-x64"
-  name        = "puppet_do.jenkins.io"
+  name        = "puppet.do.jenkins.io"
```

Comment:
Since the value duplicates with the hostname passed to the user_data, do you mind using a `local` (in a subsequent PR: let's validate the change here)